aiida/sphinxext/tests/conftest.py

```diff
@@ -7,47 +7,58 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
+"""Pytest fixtures for AiiDA sphinx extension tests."""
 from __future__ import division
 from __future__ import print_function
 from __future__ import absolute_import
 import os
+from os import path
 import sys
 import shutil
 import tempfile
 import subprocess
-from os.path import join, dirname
 import xml.etree.ElementTree as ET
 
 import pytest
 
+
 @pytest.fixture
 def reference_result():
+    """Return reference results (for check)."""
+
     def inner(name):
-        return join(dirname(__file__), 'reference_results', name)
+        return path.join(path.dirname(__file__), 'reference_results', name)
+
     return inner
 
+
 @pytest.fixture
 def build_dir():
+    """Create directory to build documentation."""
     # Python 2 doesn't have tempfile.TemporaryDirectory
     dirname = tempfile.mkdtemp()
     try:
         yield dirname
-    except Exception as e:
-        raise e
+    except Exception as exc:
```

Comment:
Is this a standard way to do this, simply raising the exception if/when it is already raised? Why not skip the `try-except` and simply have the `yield` and then the `shutil` line?

Reply:
yeah, the try/except is kind of pointless here.